<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVerwaltungClass</class>
 <widget class="QMainWindow" name="QVerwaltungClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QVerwaltung</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="cb_choose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Objekte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mieter</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="tb_search">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_overview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>831</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QVerwaltung.qrc"/>
 </resources>
 <connections/>
</ui>
